src/components/layoutStyles.css

```diff
@@ -0,0 +1,42 @@
+/* global.css */
+
+body {
+  --bg: #fff;
+  /* OC Gray 9 */
+  --textNormal: #212529;
+  --textTitle: #212529;
+  /* OC Blue 6 */
+  --textLink: #228be6;
+  --hr: hsla(0, 0%, 0%, 0.2);
+
+  background-color: var(--bg);
+}
+
+body.dark {
+  -webkit-font-smoothing: antialiased;
+  /* OC Grey 9 */
+  --bg: #212529;
+  --textNormal: #dbe4ff;
+  --textTitle: #fff;
+  --textLink: #f03e3e;
```

Comment:
could we keep the link the same as with the light theme? and maybe adjust the blue to 5 or something..

Reply:
No need to change the link colors ideally.

Reply:
Sounds good, lemme send a SS real quick though

Reply:
So this is if the dark theme links are kept as the light theme links currently are:
![image](https://user-images.githubusercontent.com/5106417/77329759-6fae5880-6cec-11ea-80c4-d6536deb49d5.png)

And this is when they are updated to the next darkest blue

![image](https://user-images.githubusercontent.com/5106417/77329942-ae441300-6cec-11ea-8649-0d1400de794b.png)

I think the contrast for the links at the bottom may not be enough, thoughts?

Reply:
Barely see a difference -- except maybe the bottom links, which are a bit too dark, true.

Reply:
If anything we need to go to the lightest blue or leave as is.